src/main/java/core/basesyntax/Node.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+public class Node<K,V> {
```

Comment:
It's better not to have class Node as a public class. It is better to keep it as a part of your map class.
```suggestion
private static class Node<K, V> {
```